lib/synth-audio.js

```diff
@@ -1119,7 +1119,7 @@ const synthCartesia = async(logger, {
         ),
         ...(opts.speed || opts.emotion && {
           experimentalControls: {
-            ...(opts.speed && {speed: opts.speed}),
+            ...(opts.speed !== null && opts.speed !== undefined && {speed: opts.speed}),
```

Comment:
I don't see how the new line should have a different result than that which it replaced?